File: beecrowd/1182.c
#include <stdio.h>

int main() {

    int i, coluna;
    float soma, matriz[144];
    char operacao;

    soma = 0;

    scanf("%d%*c", &coluna);
    scanf("%c", &operacao);

    for(i = 0; i < 144; i++) {
        scanf("%f", &matriz[i]);
        if ((i % 12) == coluna) {
            soma += matriz[i];
        }
    }
    
    if(operacao == 'S') {
        printf("%.1f\n", soma);
    }else {
        printf("%.1f\n", soma/12);
    }
    return 0;
}

File: beecrowd/1182.py
coluna = int(input())
operacao = input()
soma = 0
for c in range(144):
    valor = float(input())
    if (c % 12) == coluna:
        soma += valor
if operacao == 'S':
    print(f'{soma:.1f}')
else:
    media = soma / 12
    print(f'{media:.1f}')